README.md

```diff
@@ -1,38 +1,19 @@
 # JavaScript Basics in Express
 
-## Getting started
-
-Ensure your Visual Studio Code application is up to date, and that you have the [ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) installed and enabled.
-
-Run the commands below inside your `Projects` folder.
-
-```bash
-git clone git@github.com:MCRcodes/javascript-basics-express.git
-cd your-project-folder-name
-npm install
-```
-
-## Copy over your JavaScript Basics code
-
-:rotating_light: **Important!**
-
-This exercise makes use of your previous `javascript-basics` code. Copy the files from that project's `src/` folder into a new `lib/` directory in this repository's `src` directory. You should end up with the following file structure in this project:
-
-```
-src
-├── app.js
-└── lib
-    ├── arrays.js
-    ├── booleans.js
-    ├── numbers.js
-    ├── objects.js
-    └── strings.js
-```
-
-## Running tests
-
-You can run tests with the `npm test` command.
-
-## Remember: READMEs are important!
-
-When you're finished, be sure to rewrite this one so it's more specific to your project.
+## Learning objectives
```

Comment:
A description of what your code actually does would be good here. While the learning objectives are useful for you, they arent a massive help to anyone who may want to use your code as theres no explicit description of what it actually does

Also some installation instructions would be great, they are a standard feature of all good READMEs